module/VuFind/src/VuFind/Resolver/Driver/PluginManager.php

```diff
@@ -49,7 +49,7 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
         '360link' => Threesixtylink::class,
         'alma' => Alma::class,
         'demo' => Demo::class,
-        'ezb' => Ezb::class,
+        'jop' => Jop::class,
```

Comment:
We should set up some aliases from ezb and the old class name to point to the new jop name, so people with outdated configurations won't break their resolving.